app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
   include DeviseTokenAuth::Concerns::SetUserByToken
-  # protect_from_forgery with: :exception
+
   protect_from_forgery with: :null_session
+  before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
why are you doing this :thinking:  ? i dont see how this is related to the book model :thinking: